modules/repository/init.go

```diff
@@ -49,29 +54,25 @@ func LoadRepoConfig() {
 	for i, t := range types {
```

Comment:
Since `LoadRepoConfig` will be invoked by `services/repository.Init`, maybe we can remove log.Fatal but return errors.

Reply:
done in 8894acfbef58ff41c325b7818a25fb0d80749b20